src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1837,23 +1864,67 @@ public void setSilentMode(boolean silentMode) {
         this.silentMode = silentMode;
     }
 
+    /**
+     * DataBoundSetter for TopicAssociation.
+     * Used for jelly file.
+     *
+     * @param topicAssociation the TopicAssociation object.
+     */
+    @DataBoundSetter
+    public void setTopicAssociation(final TopicAssociation topicAssociation) {
+        this.topicAssociation = topicAssociation;
+    }
+
+    /**
+     * Enable or disable Topic Association option.
+     *
+     * @param enable true or false.
+     */
+    public void setEnableTopicAssociation(boolean enable) {
+        if (this.topicAssociation == null) {
+            this.topicAssociation = new TopicAssociation();
+        }
+        this.topicAssociation.setEnabled(enable);
+    }
+
     /**
      * Check if topic association is enabled.
+     * Used for jelly file.
+     *
      * @return true if so.
      */
-    public boolean isEnableTopicAssociation() {
```

Comment:
old method signatures should be kept as much as possible for backwards binary compatibility.